editoast/src/models/stdcm_log.rs

```diff
@@ -0,0 +1,23 @@
+use editoast_derive::Model;
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+use crate::core::stdcm::Request;
+use crate::core::stdcm::Response;
+
+editoast_common::schemas! {
+    StdcmLog,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, Model, ToSchema)]
+#[model(table = editoast_models::tables::stdcm_logs)]
+#[model(gen(ops = cru, list))]
```

Comment:
```suggestion
#[model(gen(ops = c))]
```

They're only created now. We can always add more code gen when necessary.

Reply:
Done.